### DISCUSSION TITLE
User with no authored commit appears in list of contributors

### DISCUSSION THREAD

**Discussion by adamoudad** [community/community#38897] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello,

I have two repositories on Github having the same issue: a github user (named "aehogan") without any authored commit appears in the "Contributors" list on the right side of the page of the repository. The repositories concerned are the following.
https://github.com/adamoudad/dot-files
https://github.com/adamoudad/tifinagh-tools

The issue appears if I forget to set up my username and email in my local git repository before commiitting and pushing my changes. For the repository https://github.com/adamoudad/dot-files , the github user in question is wrongly associated to a commit I authored as "Adam Oudad <adam@localhost.localdomain>".
For the repository https://github.com/adamoudad/tifinagh-tools , I actually fixed all the commit with the right author, but the other github user still appears in the "Contributors" list, although the user does not appear under the detailed contributions page https://github.com/adamoudad/tifinagh-tools/graphs/contributors

How can I remove the unrelated github user from the list of contributors?

Thanks.
Adam.

**Comment by jfullstackdev** [comment]
yup, there are similar issues as well, but that's depending on the situation

first, it can happen when you have two accounts in the same computer

second, wrong commit setup

you can give further details so we can take a look too

**Comment by adamoudad** [comment]
Hi @jdevstatic , thank you for your answer.

What details are you interested in?

Concerning the issue on the repository https://github.com/adamoudad/tifinagh-tools , it is related to the newest commit which I amended, and is now replaced by fixed commit 78873770f070026d6b6d59e95b84c0a5f01a5bcc [(link)](https://github.com/adamoudad/tifinagh-tools/commit/78873770f070026d6b6d59e95b84c0a5f01a5bcc)

Concerning the issue on the repository https://github.com/adamoudad/dot-files , it is precisely related to this commit `a8785398ac42731257703665750bfe4ccf0ba804` [(link)](https://github.com/adamoudad/dot-files/commit/a8785398ac42731257703665750bfe4ccf0ba804).

**Comment by airtower-luna** [comment, marked as answer]
> For the repository https://github.com/adamoudad/tifinagh-tools , I actually fixed all the commit with the right author, but the other github user still appears in the "Contributors" list, although the user does not appear under the detailed contributions page https://github.com/adamoudad/tifinagh-tools/graphs/contributors

That sounds like a caching issue. You can try to [contact GitHub support](https://support.github.com/request) and ask them to clear that cache and rebuild the contributor list.
